include/wabt/ir.h

```diff
@@ -1252,6 +1252,7 @@ struct Module {
   struct {
     bool simd = false;
     bool exceptions = false;
+    bool atomic_ops = false;
```

Comment:
Any chance we can keep these names in sync with the names in feature.def (which here would be `threads`)?